docs/source/supercheq/Supercheq_css.ipynb

```diff
@@ -5,28 +5,57 @@
    "id": "7769945b",
```

Comment:
<p>Is it ok to put this button and the next in the same cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>